subsys/bluetooth/controller/ll_sw/ull_iso.c

```diff
@@ -143,6 +143,7 @@ void ll_iso_tx_mem_release(void *node_tx);
 
 #define ISO_TEST_TX_BUFFER_SIZE 32U
 
+#if BT_CTLR_ISO_TX_BUFFERS
```

Comment:
We do have Kconfig related to vendor specific path? can we use it here as conditional compile? I am not liking use of numeric value for conditional compile.

Reply:
We would still want to have vendor- and HCI path active in the code, for testing

Reply:
AS `BT_CTLR_ISO_TX_BUFFERS` is not a `CONFIG_XXX`, it is always defined, so the risk of checking value of an undefined macro is eliminated. Can we do a composite define like
`#define ISO_USE_TX_BUFFERS (BT_CTLR_ISO_TX_BUFFERS > 0) && defined(CONFIG_BT_CTLR_ISO_VENDOR_DATA_PATH))`